sdk/core/core-http/package.json

```diff
@@ -60,14 +69,15 @@
   },
   "scripts": {
     "audit": "node ../../../common/scripts/rush-audit.js && rimraf node_modules package-lock.json && npm i --package-lock-only 2>&1 && npm audit",
-    "build": "run-s build:tsc build:rollup",
+    "build": "run-s build:tsc build:types build:rollup",
     "build:scripts": "tsc -p ./.scripts/",
     "build:test": "run-s build build:test-browser",
     "build:tsc": "tsc -p tsconfig.es.json",
+    "build:types": "downlevel-dts types/ types3.1/",
     "build:rollup": "rollup -c 2>&1",
     "build:test-browser": "rollup -c rollup.test.config.js 2>&1",
     "check-format": "prettier --list-different --config ../../.prettierrc.json \"src/**/*.ts\" \"test/**/*.ts\" \"*.{js,json}\"",
-    "clean": "rimraf ./es ./dist ./dist-test",
+    "clean": "rimraf ./es ./dist ./dist-test ./dist-browser ./types ./types3.1",
```

Comment:
should we nest all these inside a `types` folder so we don't have to grow this list as we support differing versions of TS? Or is that a problem for future packages?

Reply:
yes, I was concerned about the relative path, but it looks like the path is adjusted correctly by typescript compiler.

Reply:
Fixed